test/Tests/src/Data/Statistics_Spec.enso

```diff
@@ -0,0 +1,132 @@
+from Standard.Base import Nothing, Vector, Number, True, Illegal_Argument_Error
+
+import Standard.Base.Data.Statistics
+from Standard.Base.Data.Statistics import all
+
+import Standard.Test
+
+# === Test Resources ===
+
+type Ord number
+
+Ord.compare_to : Ord -> Ordering
+Ord.compare_to that = this.number.compare_to that.number
```

Comment:
I'd suggest making the compare to non-trivial, like reversing the order. Thus it will more robustly check that we are actually using this one and not some default from somewhere.